### DISCUSSION TITLE
Will Towny + Towny Dynmap ever support Forge?

### DISCUSSION THREAD

**Discussion by DNAmaster10** [TownyAdvanced/Towny#5047] [Q&A] [answered]
Im currently in the process of migrating from a Paper MC server to a Forge server, and was wondering if there was a way to use Towny and Dynmap Towny with Dynmap Forge. My server relies heavily on the displaying of borders on the Dynmap, and having an  admin drawing area markers would be extremely tedious.

**Comment by LlmDl** [comment, marked as answer]
I don't think I have any plans to do so. I am already incredibly underpaid when it comes to what I put into Towny. Supporting another server platform is probably unrealistic for me. Towny is a great hobby but it doesn't pay well.

**Reply by LlmDl** [reply]
Maybe look into the Forge + Bukkit options like mohist (and others I don't remember the name of.)

**Reply by DNAmaster10** [reply]
Of course! Would like to mention that Towny in its current state has been of incredible use to me, and it truly is an amazing plugin. Keep up the amazing work, and i hope you manage to find a better source of pay in future.

**Reply by Semisol** [reply]
Just saw this and wanted to say keep up the work.
